go/common/update.go

```diff
@@ -283,15 +285,16 @@ func UpdateFromBytes(data []byte) (Update, error) {
 
 	// Read list of balance updates
 	if balancesSize > 0 {
-		if len(data) < int(balancesSize)*(len(Address{})+len(Balance{})) {
+		balanceLength := len(amount.Amount{}.Bytes32())
```

Comment:
nit: this is a long form of writing
```
balanceLength := 32
```
in which case it could also be written as
```suggestion
		const balanceLength = 32
```

Reply:
I wanted to avoid "magic" constants, but in this case it might sense.